### DISCUSSION TITLE
Apiman 3.1.0.Final is out!

### DISCUSSION THREAD

**Discussion by msavy** [apiman/apiman#2495] [Announcements]
I'm delighted to announce that I have released Apiman 3.1.0.Final.

You can get it from the [usual places](https://www.apiman.io/download.html) (or GitHub releases, it's the same!).

Aside from numerous bug fixes and a few interesting new features, this includes a security fix for [CVE-2023-28640](https://www.apiman.io/blog/potential-permissions-bypass-disclosure/). Thanks to @volkflo and @BastianGem for responsible disclosure of this issue.

Note: Due to an issue with the release pipeline, we ended up having to cut a 3.1.1.Final release also, but it’s identical to 3.1.0.Final.

Read more here — https://www.apiman.io/blog/apiman-3.1.1.Final/

**Comment by carlomarco** [comment]
Hello! I would like to start testing the Apiman 3.1.1 but the quickstart docker compose download link is broken.

**Reply by msavy** [reply]
Thanks for letting me know, @carlomarco — I have updated the website to reflect the recent monorepo refactoring changes. It should update within a few minutes, but might require a force refresh in your browser.

**Reply by carlomarco** [reply]
Working now, thanks Marc!
